astro/src/content/docs/get-started/start-here/index.mdx

```diff
@@ -0,0 +1,205 @@
+
+<StartDiagram alt="Launch, then delegate, then add functionality." />
+
+Let's walk through an example where you'll add a Login With Google button to an application with no code changes required.
+
+## Prerequisites
+
+* git
+* Docker (if you want to follow the Docker instructions)
+* a GitHub account (if you choose to use GitPod instead)
+* node (this was tested with `v22.12.0`)
+
+## Launch FusionAuth
+
+FusionAuth is downloadable software, so you need to run it somewhere. For this guide, you can either:
+
+* Use Docker to install and run FusionAuth and the application
+* Use GitPod to run FusionAuth in the cloud in your GitPod account, with no software install needed.
+
+Expand the section you want to follow and get started.
+
+### Install and Run in Docker
+<details>
+<summary>Docker</summary>
+<DockerInstallAndStart codeRoot={frontmatter.codeRoot} kickstartUri={frontmatter.codeRoot + "/kickstart/kickstart.json"}/>
+</details>
+
+### Install and Run in GitPod
+<details>
+<summary>GitPod</summary>
+<GitPodStart codeRoot={frontmatter.codeRoot} kickstartUri={frontmatter.codeRoot + "/kickstart/kickstart.json"}/>
+</details>
+
+### What Did You Just Do?
+
+You launched a FusionAuth database and server either using Docker or in GitPod, using a kickstart to pre-populate the application details in the FusionAuth database.
+
+## Delegate Authentication
+
+Once you have FusionAuth up and running, you need to set up your application or applications to delegate to FusionAuth. For this guide, there's an example `Start Here` application which is already set up to do so. Let's look at that configuration and then start it up so you can log in.
+
+### Open the Admin Interface
+
+* In the previous step, you logged in to the system as an admin user.  Go back to that browser screen, which should be showing the administrative interface for FusionAuth.
+* Navigate to <InlineUIElement>Applications</InlineUIElement> in the left-hand sidebar. You may need to use the hamburger icon at the top left of the screen to see all options.
+* Choose the `Start Here` application.
+* Choose <InlineUIElement>Edit</InlineUIElement> from the <InlineUIElement>Actions</InlineUIElement> dropdown.
+
+### Application Configuration
+
+After the user signs in at FusionAuth, they need to get back to your application. In the FusionAuth application configuration you can view authorized redirect URLs and how tokens are generated/exchanged.
+
+![The edit screen of the application.](/img/docs/get-started/start-here/edit-application.png)
+
+### After the User Authenticates
+
+To understand how the JavaScript application integrates with FusionAuth, take a quick look at some of the code from the repository.
+
+We craft a URL that includes the `clientId` from the configuration you reviewed above, as well as some other parameters. This is where the user goes to log in with FusionAuth. TODO need to only show the `redirect(302, `fusionAuthURL/oauth2/authorize` line
```

Comment:
Need to update the code excerpt (the tags in the repo files) and remove the TODO here.

Reply:
I'm not sure what you mean here, need more context.

Reply:
Oh, I get it, you want a *different* code block which is smaller than what you have.  I will do that now.

Reply:
I took care of these, per our discussion.